entries/s/shift4shop.com.json

```
@@ -0,0 +1,12 @@
+{
+  "Shift4Shop": {
+    "domain": "shift4shop.com",
+    "tfa": [
+			"totp"
+		],
+		"documentation": "https://www.shift4shop.com/lp/features/2-factor-authentication/",
```

```suggestion
		"documentation": "https://support.3dcart.com/knowledgebase/article/View/866/11/",
```
I think it's probably better to use this as the documentation link. The current one links to this one as a "Knowledgebase Article" and this one provides far greater detail and clarity than the current one.